_posts/2020-12-23-infinispan-12.0.0.CR1.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: blog
+title: Infinispan 12.0.0.CR1
+permalink: /blog/:year/:month/:day/infinispan-12
+date: '2020-23-12T12:00:00.000-00:00'
+author: ttarrant
+tags: [ "release", "development" ]
+---
+
+= Infinispan 12.0.0.Dev03
```

Comment:
Wrong version

Reply:
well we can publish this with this we release better ? with all the stuff